warehouse/cli/search/reindex.py

```diff
@@ -57,6 +57,8 @@ def reindex(config, **kwargs):
     """
     client = config.registry["elasticsearch.client"]
     db = Session(bind=config.registry["sqlalchemy.engine"])
+    number_of_replicas = config.registry.get("elasticsearch.replicas", 0)
+    refresh_interval = config.registry.get("elasticsearch.interval", "5s")
```

Comment:
Simply because this is what it had been set to in the original PR: https://github.com/pypa/warehouse/pull/1226/files#diff-80840db14f707f8d7c8db4ee49bfd4efR110

It seems like the default is actually "1s": https://www.elastic.co/guide/en/elasticsearch/reference/2.4/index-modules.html#dynamic-index-settings

I don't see any reason why it shouldn't go back to the default version, unless @robhudson has a reason.